src/game-logic/reducers/purchaseSmelter.js

```diff
@@ -5,7 +5,6 @@ import { FORGE_AVAILABLE_NOTIFICATION } from '../../strings'
 import { showNotification } from './showNotification'
 import { updateLearnedRecipes } from './updateLearnedRecipes'
 
-// TODO: Add tests for this reducer
```

Comment:
i had copied this file as my starting point for purchaseComposter so i went back and added the specs for this as well